client/web/web.go

```diff
@@ -532,30 +550,36 @@ func (s *Server) serveGetNodeData(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
+	cv, err := s.lc.CheckUpdate(r.Context())
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
```

Comment:
nit: maybe log error but don't prevent the node data response from completing

could go either way for this, but since it's the only part that could potentially impact our legacy client, it's probably worth erring on the safer side

Reply:
oh, true